src/status_im2/common/toasts/view.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im2.common.toasts.view
+  (:require
+   [quo2.core               :as quo2]
```

Comment:
we just use `quo` under status-im2 namespace

Reply:
fixed